test/drop_test/ffi.py

```diff
@@ -11,6 +11,11 @@
 
 DEBUG_PRINT_EVENT = True
 
+# fmt: off
+PRIV_KEY = bytes([164, 70, 230, 247, 55, 28, 255, 147, 128, 74, 83, 50, 181, 222, 212, 18, 178, 162, 242, 102, 220, 203, 153, 161, 142, 206, 123, 188, 87, 77, 126, 183])
+PUB_KEY = bytes([68, 103, 21, 143, 132, 253, 95, 17, 203, 20, 154, 169, 66, 197, 210, 103, 56, 18, 143, 142, 142, 47, 53, 103, 186, 66, 91, 201, 181, 186, 12, 136])
+# fmt: on
+
```

Comment:
This currently uses the same private/public pair for all the peers so it doesn't fully exercise that the signing actually works. It should be several separate keypairs for each runner if I got it right. Keypairs could be hardcoded in `test/run.py` for each runner

Reply:
Yeah, that's a fair point. Will try to implement that

Reply:
Done